sky/services/platform/interface.mojom

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module platform_interface;
+
+/// Specifies an acceptable orientation for the interface to be in
+/// A mask of acceptable interface orientations is created by or'ing individual
+/// options
+enum /* options */ InterfaceOrientation {
+  Portrait  = 1,
+  Landscape = 2,
+};
+
+/// Specifies a particular device orientation
+/// A mask of device interface orientations is created by or'ing individual
+/// options
+enum /* options*/ DeviceOrientation {
+  PortraitUp      = 1,
+  PortraitDown    = 2,
+  LandscapeLeft   = 4,
+  LandscapeRight  = 8,
+};
+
+/// Specifies how the status bar(s) should be displayed
```

Comment:
This is very hard to be generic about. Android, Chrome OS, Windows, Mac OS, iOS, they're all different. Every window manager on X11 has a different idea too. But maybe:

```
enum SystemUIOverlays {
  Top      = 1,
  Bottom   = 2,
  Floating = 4,
  All      = 0xFFFFFFFFFFFFFFFF
}
```

...and you or them together to say which should be hidden.